crypto/bn254/keystore/keystore.go

```diff
@@ -0,0 +1,32 @@
+package keystore
+
+import (
+	"context"
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/types"
+)
+
+type Signer struct {
+	keyPair *bls.KeyPair
+}
+
+func New(path, password string) (*Signer, error) {
+	keyPair, err := bls.ReadPrivateKeyFromFile(path, password)
+	if err != nil {
+		return nil, err
+	}
+	return &Signer{
+		keyPair: keyPair,
+	}, nil
+}
+
+func (s *Signer) SignMessage(ctx context.Context, message [32]byte) ([]byte, error) {
+	signatureBytes := s.keyPair.SignMessage(message).Bytes()
+	var signature []byte
+	copy(signature[:], signatureBytes[:])
```

Comment:
why do you have to copy like this? signatureBytes is a fixed array [32]byte or something?

Reply:
> signatureBytes is a fixed array [32]byte
Yep that's exactly why I need to copy.